src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method name `getStatistic` does not clearly convey that it performs file reading and writing operations. Consider renaming it to better reflect its functionality, following the checklist item 'Stick to the method naming convention'.